features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -957,18 +1056,124 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
+                                    fetchedAttributes
+                                        ?.filter((claim: Claim) => {
+                                            return !claim.properties?.some(
+                                                (property: Property) => property.key === "isSystemClaim" &&
+                                                    property.value === "true"
+                                            );
+                                        })
+                                        .map((claim: Claim) => ({
+                                            text: claim.claimURI,
+                                            value: claim.claimURI
+                                        }))
+                                        .sort(
+                                            (a: { text: string }, b: { text: string }) =>
+                                                a.text.localeCompare(b.text)
+                                        )
+                                }
+                                onChange={ (
+                                    event: React.SyntheticEvent<HTMLElement, Event>,
+                                    data: { value: string }
+                                ) => {
+                                    if (event.type === "click" && !subAttributes.includes(data.value)) {
+                                        setSubAttributes([ ...subAttributes, data.value ]);
+                                    }
+                                } }
+                                data-testid={ `${testId}-form-sub-attributes-dropdown` }
+                                search
+                            />
+                            <div>
+                                { subAttributes.map((attribute: string, index: number) => (
+                                    <div
+                                        style={ { alignItems: "center", display: "flex",
+                                            justifyContent: "space-between"
+                                        } }
+                                        key={ index }>
+                                        <span>{ attribute }</span>
+                                        <IconButton
+                                            disabled={ isReadOnly }
+                                            onClick={ () => {
+                                                setSubAttributes(
+                                                    subAttributes.filter(
+                                                        (item: string) => item !== attribute
+                                                    )
+                                                );
+                                            } }
+                                            data-componentid={ `${testId}-delete-sub-attribute-${index}` }
+                                            style={ { marginLeft: "auto" } }
```

Comment:
let's move this as well to a scss file

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/8111/commits/fc575885da12bc341fd55470310378ecbc3dc6ad